src/icalendar/tests/test_fixed_issues.py

```diff
@@ -185,54 +185,6 @@ def test_issue_101(self):
         org_cn = cal.walk('VEVENT')[0]['ORGANIZER'].params['CN']
         self.assertEqual(org_cn, 'acme, ädmin')
 
-    def test_issue_104__ignore_exceptions(self):
-        """
-        Issue #104 - line parsing error in a VEVENT
-        (which has ignore_exceptions). Should mark the event broken
-        but not raise an exception.
-        https://github.com/collective/icalendar/issues/104
-        """
-        ical_str = """
-BEGIN:VEVENT
-DTSTART:20140401T000000Z
-DTEND:20140401T010000Z
-DTSTAMP:20140401T000000Z
-SUMMARY:Broken Eevnt
-CLASS:PUBLIC
-STATUS:CONFIRMED
-TRANSP:OPAQUE
-X
-END:VEVENT"""
-        event = icalendar.Calendar.from_ical(ical_str)
-        self.assertTrue(isinstance(event, icalendar.Event))
-        self.assertTrue(event.is_broken)  # REMOVE FOR NEXT MAJOR RELEASE
```

Comment:
Uh, this is interesting .. a TODO for the next Major release which we are drafting